web3/contract.py

```diff
@@ -637,9 +686,16 @@ class ConciseContract:
     @deprecated_for(
         "contract.caller.<method name> or contract.caller({transaction_dict}).<method name>"
     )
-    def __init__(self, classic_contract, method_class=ConciseMethod):
-
-        classic_contract._return_data_normalizers += CONCISE_NORMALIZERS
+    def __init__(
+        self,
+        classic_contract: Contract,
+        method_class: Union[Type['ConciseMethod'], Type['ImplicitMethod']]=ConciseMethod
+    ) -> None:
+        # web3/contract.py:711: error: Incompatible types in assignment (expression has type
+        # "Tuple[Callable[..., Any], Callable[..., Any]]", variable has type
+        # "Tuple[Callable[..., Any]]")
+        # type ignored b/c error inference +=
+        classic_contract._return_data_normalizers += CONCISE_NORMALIZERS  # type: ignore
```

Comment:
It seems to me like mypy is having some trouble with the `+=` here: I can't figure out how to resolve the error it throws: `web3/contract.py:711: error: Incompatible types in assignment (expression has type "Tuple[Callable[..., Any], Callable[..., Any]]", variable has type "Tuple[Callable[..., Any]]")` - which is really strange since it's just repeating the same type twice in the first definition.

Reply:
When you use a `Tuple` like a collection like e.g. `(1, 2, 3,)` there are multiple types you could use to describe that. 

1. 

```python
Tuple[int, int, int]
```

2.

```python
Tuple[int, ...]
```

What happens in your example is that mypy infers different types for the variable `_return_data_normalizers`  and the expression. If you add a type hint so that mypy knows that it should treat `_return_data_normaliers` as `Tuple[Callable[..., Any], ...]` the error should go away.

Reply:
Ahhh, finally got things worked out - thanks @cburgdorf!!